sensor_msgs/Image image
int32 frame_size